build/helper/parameter_usage_options.py

```diff
@@ -14,6 +14,8 @@ class ParameterUsageOptions(AutoNumber):
 
     SESSION_METHOD_DECLARATION = ()
     '''For declaring a regular method in Session'''
+    SESSION_METHOD_DECLARATION_NO_DEFAULT = ()
+    '''Same as SESSION_METHOD_DECLARATION but without default values - used for passing through parameters to the same function, just generated elsewhere'''
```

Comment:
One source of confusion for me was that this is used for rendering code at _call site_, not at "method declaration" site.

So something like `SESSION_METHOD_CALL` or `SESSION_METHOD_PASSTHROUGH_CALL` would be much better (see: `LIBRARY_METHOD_CALL`)

For docstring:

> For calling into Session using parameters of the same name that are simply passed through.

Reply:
``` python
    SESSION_METHOD_CALL = ()
    '''Same as SESSION_METHOD_DECLARATION but without default values - For calling into Session using parameters of the same name and order that are simply passed through.'''
```
It is useful to know it is the same as `SESSION_METHOD_DECLARATION` so I left that part in